react-express-mysql/docker-compose.yaml

```diff
@@ -28,7 +28,10 @@ services:
     depends_on:
       - db
   db:
-    image: mysql:8.0.19
+    # We use a mariadb image which support both amd64 & arm64 architecture
+    image: mariadb:10.6.4-focal
```

Comment:
Should we add the DB healthcheck to improve a bit ? A few samples have it already 😁

Reply:
I prefer to have a dedicated PR for DB healthcheck, I saw some errors when testing the samples and I think that it'll be better to spend some time completely focus on the healthcheck